onnxruntime/core/framework/session_state.cc

```diff
@@ -185,6 +249,20 @@
   return &p_seq_exec_plan_.value();
 }
 
+void Print(const std::string& type, gsl::span<const AllocPlanPerValue> alloc_plans) {
+  std::cout << type << std::endl;
```

Comment:
>std::cout << type << std::endl; [](http://example.com/codeflow?start=2&length=31)

It would be more practical to accept an argument to std::ostream& so any kind of stream can be used.

Reply:
agree. actually we can remove those piece of code now. there is a better support of dump allocation plan in allocation_planner.cc, so we can remove this one.